### DISCUSSION TITLE
I've made a guide for a minimalist SO self-hosted deployment for personal or small business use. Feedback is highly appreciated!

### DISCUSSION THREAD

**Discussion by mendo1024** [Security-Onion-Solutions/securityonion#9714] [General]
This is SIEM monitoring of a Nextcloud instance, but it could be anything. I hope it will help someone curious with basic admin experience.

[How to secure the self-hosted Nextcloud](https://cybrainz.io/blog/securing_nextcloud)

Please check it out and let me know what you think.

**Comment by innovate-support** [comment]
I like the detail you put into that. It's essentially how I was using SO in our environment when it was first released. I have since moved to a more diverse distributed layout, and even use mini PC's as remote sensors for small businesses and home users.
